### ISSUE TITLE
LocalizationProvider does not expose children prop for React 18 typescript types

### ISSUE THREAD

**Issue by thebetternewt** [telerik/kendo-react#1275]
## I'm submitting a...

* Bug report 

## Current behavior
`<LocalizationProvider>` component from `@progress/kendo-react-intl` does not expose a `children` prop to be compatible with `@types/react` version `18.0.0`.

See https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-typescript-definitions

## Expected behavior
`<LocalizationProvider>` component from `@progress/kendo-react-intl` should expose an explicit `children` prop.

## Minimal reproduction of the problem with instructions

```typescript
// inside React component...
<LocalizationProvider language="single-file">
  {children}
</LocalizationProvider>
```

```
// Error:
No overload matches this call.
  Overload 1 of 2, '(props: LocalizationProviderProps | Readonly<LocalizationProviderProps>): LocalizationProvider', gave the following error.
    Type '{ children: Element; language: string; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<LocalizationProvider> & Pick<Readonly<LocalizationProviderProps>, "language"> & Pick<...> & Pick<...>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<LocalizationProvider> & Pick<Readonly<LocalizationProviderProps>, "language"> & Pick<...> & Pick<...>'.
  Overload 2 of 2, '(props: LocalizationProviderProps, context: any): LocalizationProvider', gave the following error.
    Type '{ children: Element; language: string; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<LocalizationProvider> & Pick<Readonly<LocalizationProviderProps>, "language"> & Pick<...> & Pick<...>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<LocalizationProvider> & Pick<Readonly<LocalizationProviderProps>, "language"> & Pick<...> & Pick<...>'.ts(2769
```

## What is the motivation or use case for changing the behavior?
Support for `@types/react` version `18.0.0+`

## Environment

Package versions:

- `@progress/kendo-react-intl@5.2.0`
- `react@18.0.0`
- `react-dom@18.0.0`
- `@types/react@18.0.0`
- `@types/react-dom@18.0.0`

Browser:

- Chrome (desktop) version 100.0.4896.75
- Edge version 97

System:
- TypeScript version: 4.6.3
- Node version: 14.17.3
- Platform:  Linux

**Comment by kdikov82** [comment]
Applicable for all components and their Props. Global solution for all Props should be introduced by adding the "children" prop

**Comment by adhip94** [comment]
Any update on the progress for React 18 compatibility? This is a showstopper for me when using the Kendo React Chart Component! Other components which I have used such as DropdownList and Upload dont have any problems with React 18.

**Comment by InaGlushkova** [comment]
@adhip94 The fix is going to be part of the next release. 
We will leave a comment with the version in which it is included.

**Comment by billyjacobs2014** [comment]
When you say next release, do you mean next major release or an upcoming minor version release? Did they change something last minute that you were unable to test?

**Comment by nstoychev** [comment]
We have published a fix in out development channel - v5.3.0-dev.202204281202. Please, see the following article about obtaining the development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

The next major release is planned for the next week.

**Comment by InaGlushkova** [comment]
Introduced with version [v5.3.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#ce8acda4-4843-57d6-be59-4e1c3f75abdc)
